src/omotes_simulator_core/infrastructure/utils.py

```diff
@@ -19,17 +19,21 @@
 from pathlib import Path
 
 
-def pyesdl_from_file(file_path: str) -> EnergySystemHandler:
+def pyesdl_from_file(file_path: str | Path) -> EnergySystemHandler:
     """
     Loads esdl file into memory and returns a handle to be able to use this file.
 
     Please note that it is not checked if the file is a valid esdl file.
-    :param file_path: string pointing to the esdl file to be loaded into memory
+    :param file_path: string or Path pointing to the esdl file to be loaded into memory
```

Comment:
Use the following format: 

:param str | Path file_path: Location of the ESDL file to be loaded into memory.